src/Store/VimStoreConnector.re

```diff
@@ -76,6 +76,12 @@ let start =
       |> List.map(c => String.make(1, c))
       |> String.concat("");
 
+    let isMultipleLines =
+      fun
+      | [s] => String.contains(s, '\n')
+      | [..._] => true
```

Comment:
What? Apparently this is valid. It doesn't do what you think it does though. It translates to just `_`. Fortunately you should get a warning that the last case is unused. It should be:

```suggestion
      | [_, ..._] => true
```
Seems like you might have discovered a new bug in Reason though. I can even kind of imagine the nasty syntactic hack that would cause this.

Reply:
Ouch, you're right! Nice catch!